libkbfs/kbfs_ops_test.go

```diff
@@ -574,8 +574,11 @@ func makeBP(id BlockID, rmd *RootMetadata, config Config,
 		ID:      id,
 		KeyGen:  rmd.LatestKeyGeneration(),
 		DataVer: config.DataVersion(),
-		Creator: u,
-		// refnonces not needed for tests until dedup is implemented
+		BlockContext: BlockContext{
+			Creator: u,
+			// refnonces not needed for tests until dedup
```

Comment:
Ouch, this comment is really old.  While you're here, do you mind changing it to something like "Explicit refnonce testing happens elsewhere"?